cfdtools/data.py

```diff
@@ -64,6 +67,9 @@ def set_nodes(self, x, y=None, z=None):
         if z:
             self._geoprop['z'] = z
 
+    def set_mesh(self, mesh):
```

Comment:
Is `set_nodes` still useful?

Reply:
this is in the case we only need node position (probes)